src/subprocess_unix.cpp

```diff
@@ -11,47 +11,30 @@
 
 UnixImpl::UnixImpl():
   m_pid(0),
-  m_running(false),
-  m_mutex(PTHREAD_MUTEX_INITIALIZER),
-  m_waitingThread()
+  m_running(false)
 {
 }
 
 UnixImpl::~UnixImpl()
 {
   kill();
 // Android has no pthread_cancel :(
```

Comment:
This comment doesn't make sense anymore.

Reply:
Removed.